### ISSUE TITLE
Webhooks fire twice when changing an "A" record

### ISSUE THREAD

**Issue by rallakwash** [peteeckel/netbox-plugin-dns#137]
**Versions**
NetBox Version: 3.7.0
NetBox DNS Version: 0.21.13
Python Version: 3.11.2

**Describe the bug**

When changing a DNS A record, webhooks fire twice, with the same content.
The only difference is in the timestamp, about 20-30 ms.

Creating, or deleting the A record doesn't cause this issue, neither does changing any other record type.

**To Reproduce**

Change an A record, doesn't matter which field.
On the webhook listener change of the A record happens twice, while the managed PTR record only once.

![image](https://github.com/peteeckel/netbox-plugin-dns/assets/35699017/68ed8e31-6c0d-4a06-afa6-21d8119256cc)

**Comment by peteeckel** [comment]
This is a duplicate of #89 and mainly a NetBox issue. I'm working on it, but it will take some time.

**Comment by rallakwash** [comment]
Thanks for the heads up, and looking at it.

**Comment by peteeckel** [comment]
There is now a NetBox issue for the underlying problem. Sorry this was taking so long, but it's really difficult to reproduce and took some time only for creating a reference environment for the core project to test.

**Comment by peteeckel** [comment]
> This is a duplicate of https://github.com/peteeckel/netbox-plugin-dns/issues/89 and mainly a NetBox issue. I'm working on it, but it will take some time.

As it turned out, it isn't ... it's a different issue, though it looked very similar until I understood the mechanics of #89.

The good news: It can be fixed without any changes to NetBox, **this** bug is my own :-)

**Comment by rallakwash** [comment]
Thanks for checking it. Tested it today, and seems to be working fine.
Thank you again.
